File: 11800.py
ones = ["Yakk", "Doh", "Seh", "Ghar", "Bang", "Sheesh"]
twice = ["Habb Yakk", "Dobara", "Dousa", "Dorgy", "Dabash", "Dosh"]

for i in range(int(input())):
    a, b = map(int, input().split())
    if b > a: a, b = b, a

    print("Case %d:" % (i + 1), end=' ')
    if a == b: print(twice[a - 1])
    else:
        if a == 6 and b == 5:
            print("Sheesh Beesh")
        else:
            print(ones[a - 1], ones[b - 1])

File: 11800.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string ones[6] = { "Yakk", "Doh", "Seh", "Ghar", "Bang", "Sheesh" };
	string twice[6] = { "Habb Yakk", "Dobara", "Dousa", "Dorgy", "Dabash", "Dosh" };
	int T; cin >> T;

	for (int i = 0; i < T; i++) {
		int a, b; cin >> a >> b;
		if (b > a) {
			int tmp = a;
			a = b;
			b = tmp;
		}

		cout << "Case " << i + 1 << ": ";
		if (a == b) cout << twice[a - 1] << endl;
		else {
			if (a == 6 && b == 5) cout << "Sheesh Beesh\n";
			else cout << ones[a - 1] << ' ' << ones[b - 1] << endl;
		}
	}
}
